utils/notification_service.py

```diff
@@ -40,6 +40,7 @@
     "run_examples_gpu": "Examples directory",
     "run_torch_cuda_extensions_gpu": "DeepSpeed",
     "run_quantization_torch_gpu": "Quantization",
+    "run_deepspeed_gpu": "DeepSpeed",
```

Comment:
the deepspeed job is actually the `run_torch_cuda_extensions_gpu` (see 2 line above).

It's a historical leftover.

Reply:
yeh it just seemed inacceptable for me in this case to name it cuda_extensions_gpu 😅 what do you suggest ?
using it nonetheless or adding more generic names ?

Reply:
I did ask myself the same question back then 😓 Happy to make it more better name. That job is actually running 

`tests/deepspeed tests/extended`

I don't know what exactly `tests/extended` test. Let's check and decide a final name next week though.

Reply:
(after I left the above comment 😅 , i just saw you change it back )

Reply:
I think it has to do with cuda compiled kernels (torch extensions), deepspeed does that as well (jit compiles optimizers for example), maybe that's why it's called that way.